src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+

```

Comment:
redundant empty line
we don't need empty lines after class declarations or method signatures